app/controllers/profile_controller.rb

```diff
@@ -3,11 +3,11 @@ def show
   end
 
   def update
-    first_name = params[:first_name]
-    last_name = params[:last_name]
+    user_params = params.require(:user)
     user = current_user
-    user.first_name = first_name
-    user.last_name = last_name
+    user.first_name = user_params[:first_name]
+    user.last_name = user_params[:last_name]
+    user.timezone = user_params[:timezone] || "Eastern Time (US &amp; Canada)"
```

Comment:
good catch, @yozlet 

The default value will now be whatever the browser timezone is, and if left empty, will default to `User::DEFAULT_TIMEZONE`.